Modules/posixmodule.c

```diff
@@ -7,18 +7,6 @@
    of the compiler used.  Different compilers define their own feature
    test macro, e.g. '_MSC_VER'. */
 
-#ifdef __APPLE__
```

Comment:
The side effect of this is to drop support for building on 10.5 and deploying to 10.4. That's such an esoteric use case that I'm in favour of doing this even in back ports to earlier versions.